.github/workflows/pr-auto-label.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: PR-auto-labeler
+
+on: 
+  pull_request:
+    types: [labeled, unlabeled, opened, edited, synchronize]
+
+permissions:
+  # Setting up permissions in the workflow to limit the scope of what it can do. Optional!
+  contents: read # the config file
+  pull-requests: write # for labeling pull requests (on: pull_request_target or on: pull_request)
+  statuses: write # to generate status
+  checks: write # to generate status
+    
+jobs:
+
+  labeler:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check Labels
+      id: labeler
+      uses: jimschubert/labeler-action@v1
+      with:
+        GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Let's add another step, which adds a comment to the PR if the `labeler` step fails

Reply:
Example PR: https://github.com/SachinSahu431/client_golang/pull/56

Added comment when `labeler` workflow fails:
![image](https://github.com/prometheus/client_golang/assets/75629410/49c5c603-db9b-49d3-9b89-8c1994f62849)